nodes/core/cv_medianBlur.py

```diff
@@ -0,0 +1,41 @@
+import uuid
+
+import bpy
+import cv2
+from ocvl.core.node_base import OCVLNodeBase, update_node, BORDER_TYPE_ITEMS
+
+class OCVLmedianBlurNode(OCVLNodeBase):
+    n_doc = "Blurs an image using the normalized box filter."
+    n_see_also = "bileteralFilter,blur,boxFilter,GaussianBlur"
+    n_requirements = {"__and__": ["src_in"]}
+
+    def set_ksize(self, value):
+        if value % 2 == 0:
+            value = value + 1
+        self["ksize_in"] = value
+
+    def get_ksize(self):
+        return self.get("ksize_in", 5)
+
+    src_in: bpy.props.StringProperty(name="src_in", default=str(uuid.uuid4()), description="Input image.")
```

Comment:
Change description for this parameter: https://docs.opencv.org/3.0-beta/modules/imgproc/doc/filtering.html?highlight=medianblu#medianblur